### PR TITLE
CSV / XLS to table kwargs to pass to pandas 

### PR DESCRIPTION
**Pull request by shostetter**
- Allows you to pass pandas keyword arguments for csv/xls to table to take advantage of skiprows etc.
- Allows query_to_csv to work with `"` encapsulated column names in PG, needed for columns with special characters

### CODE REVIEW

**Comment by shostetter** [comment]
I believe this works though was getting some failures in unrelated areas, rerunning sometimes fixed them so not sure whats going on

File: pysqldb3/pysqldb3.py [hunk 1]
```diff
@@ -1195,133 +1207,36 @@ def xls_to_table(self, input_file=None, sheet_name=0, overwrite=False, schema=No
...
-                # Match previous styles
-                cols = []
-                for c in sheet_data[0]:
-                    if type(c) == str:
-                        cols.append(c.strip().replace(' ', '_').replace('.', '_'))
-                    else:
-                        cols.append(c)
 
-                df = pd.DataFrame(sheet_data[1:], columns=cols)
+        try:
```
Comment (chriscah) [inline pysqldb3/pysqldb3.py:1220]: We can get rid of most of the code here but maybe large tables should still default to _bulk_xlsx_to_table

**Review by chriscah — COMMENTED** [review]
a few small comments

File: pysqldb3/pysqldb3.py [hunk 2]
```diff
@@ -1228,15 +1228,37 @@ def xls_to_table(self, input_file=None, sheet_name=0, overwrite=False, schema=No
...
+                    df.to_csv(temp_file, index=False, header=True)
+
+                    success = self._bulk_csv_to_table(input_file=temp_file, schema=schema, table=table,
+                                                      table_schema=table_schema, days=days)
+                    os.remove(temp_file)
+
+                    if not success:
+                        raise AssertionError('Bulk file loading failed.'.format(schema, table))
+
+                except SystemExit:
```
Comment (chriscah) [inline pysqldb3/pysqldb3.py:1251]: if os.remove doesnt error if the file doesnt exist, could always try to remove in except as well if a problem happens midflight

**Review by chriscah — APPROVED** [review]
most recent changes look good!

**Pull request merged**